### PR TITLE
Hylics 2: Implement new game

### PR DESCRIPTION
**Pull request by TRPG0**
## What is this fixing or adding?
Hylics 2 as a new Archipelago randomizer game.

## How was this tested?
Many private tests and/or solo runs plus a couple of public tests in the discord server. I plan to do public beta testing more often now that I've finished adding extra features.

### CODE REVIEW

File: worlds/hylics2/__init__.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                .place_locked_item(self.add_item(gestures[200685]["name"], gestures[200685]["classification"], 200685))
+
+        elif self.world.gesture_shuffle[self.player] == 1: # TVs only
+            gestures = list(Items.gesture_item_table.items())
+            tvs = list(Locations.tv_location_table.items())
+
+            # if Extra Items in Logic is enabled place CHARGE UP first and make sure it doesn't get 
+            # placed at Sage Airship: TV
+            if self.world.extra_items_in_logic[self.player]:
+                tv = random.choice(tvs)
```
Comment (PoryGone) [inline worlds/hylics2/__init__.py:130]:
```suggestion
                tv = self.world.random.choice(tvs)
```

File: worlds/hylics2/__init__.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+                gest = gestures.index((200681, Items.gesture_item_table[200681]))
+                while tv[1]["name"] == "Sage Airship: TV":
+                    tv = random.choice(tvs)
```
Comment (PoryGone) [inline worlds/hylics2/__init__.py:133]:
```suggestion
                    tv = self.world.random.choice(tvs)
```

File: worlds/hylics2/__init__.py [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+                self.world.get_location(tv[1]["name"], self.player)\
+                    .place_locked_item(self.add_item(gestures[gest][1]["name"], gestures[gest][1]["classification"], 
+                    gestures[gest]))
+                gestures.remove(gestures[gest])
+                tvs.remove(tv)
+
+            for i in range(len(gestures)):
+                gest = random.choice(gestures)
```
Comment (PoryGone) [inline worlds/hylics2/__init__.py:141]:
```suggestion
                gest = self.world.random.choice(gestures)
```

File: worlds/hylics2/__init__.py [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+                tv = random.choice(tvs)
```
Comment (PoryGone) [inline worlds/hylics2/__init__.py:142]:
```suggestion
                tv = self.world.random.choice(tvs)
```

**Review by PoryGone — CHANGES_REQUESTED** [review]
Did a light readthrough, mostly looks good. Just picked out a few instances of something that needs fixing.

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**